src/Guesser/TypeGuesserInterface.php

```diff
@@ -30,4 +30,9 @@ interface TypeGuesserInterface
      * @phpstan-param class-string $class
      */
     public function guessType($class, $property, ModelManagerInterface $modelManager);
+
+    /**
+     * @phpstan-return array<TypeGuesserInterface>
+     */
+    public function getGuessers(): array;
```

Comment:
It's not needed to add this to the interface, just to the TypeGuesserChain.

Reply:
It even can not be. Other class are implementing it, and it will BC-break.